app/views/course_reviews/partials/_reviews_course_search_form.html.erb

```diff
@@ -62,9 +62,35 @@
       </div>
       -->
       <div class="field">
-        <%= f.button "Search", :class => "button is-link search-button margin-bottom-0 is-medium is-info",
-            :data => { disable_with: "<i class='fa fa-spinner fa-spin'></i>&nbspLoading".html_safe} %>
+        <%= f.button "Search", id: "search_button", :class => "button is-link search-button margin-bottom-0 is-medium is-info" %>
+      </div>
+      
+      <!-- Sort button -->
+      <div class="field">
+        <%= f.button "Sort by Rating", id: "sort_button", :name => "sort_button", :class => "button is-link search-button margin-bottom-0 is-medium is-info" %>
```

Comment:
perhaps we change the button class to button `is-outlined`, so its not as bold as the search but more subtle

Reply:
Great suggestion! I have added it